types/entity.go

```diff
@@ -19,8 +19,15 @@ const (
 
 	// EntityBackendClass is the name of the class given to backend entities.
 	EntityBackendClass = "backend"
+
+	// Redacted is filled in for fields that contain sensitive information
+	Redacted = "REDACTED"
 )
 
+// DefaultRedactFields contains the default fields to redact
+var DefaultRedactFields = []string{"password", "passwd", "pass", "api_key",
```

Comment:
They all seem sensible, default field names to redact to me, except maybe for `pass`: it sounds more like a pass/fail type of field to me, holding a boolean.

Reply:
These are already established in the product and documented. I don't want to change them in this PR.